examples/infill/infill.cpp

```diff
@@ -235,7 +235,12 @@ int main(int argc, char ** argv) {
 
     std::vector<llama_token> embd_inp;
     std::vector<llama_token> inp_pfx = ::llama_tokenize(ctx, params.input_prefix, false);
+    params.input_suffix.erase(0, params.input_suffix.find_first_not_of(" "));
     std::vector<llama_token> inp_sfx = ::llama_tokenize(ctx, params.input_suffix, false);
+    const int space_token = 29871;
+    if (params.escape && inp_sfx.size() > 1 && inp_sfx[0] == space_token) {
```

Comment:
I think we always want to remove the leading space, regardless if the `params.escape` is set or not:

```suggestion
    if (inp_sfx.size() > 1 && inp_sfx[0] == space_token) {
```

Reply:
I think we need the params.escape check as only then we add an extra space to the suffix.